pkg/registry/core/persistentvolume/strategy.go

```diff
@@ -143,7 +143,7 @@ func (persistentvolumeStatusStrategy) GetResetFields() map[fieldpath.APIVersion]
 	return fields
 }
 
-var nowFunc = metav1.Now
+var NowFunc = metav1.Now
```

Comment:
How about adding a comment explaining the reason why the function is defined, e.g. `for a test purpose.`

Reply:
This function is not entirely for testing, I just exported it to allow override in tests. It is used to set actual timestamp in PV strategy: https://github.com/kubernetes/kubernetes/blob/0fcfea649cda8915b1ceaa886eeea23d903afb55/pkg/registry/core/persistentvolume/strategy.go#L74